### ISSUE TITLE
(parent game-object) doesn't work for the root game objects

### ISSUE THREAD

**Issue by LispEngineer** [arcadia-unity/Arcadia#189]
```
minimal.core=> (object-named "Main Camera")
#unity/GameObject 9790
minimal.core=> (parent (object-named "Main Camera"))
System.NullReferenceException: A null value was found where an object instance was required.
  at arcadia/core$eval__43591$fn__43611__43615.invoke (System.Object ) [0x00000] in <filename unknown>:0 
  at arcadia/core$eval__43362$fn__43466$G__43327__43479__43482.invoke (System.Object ) [0x00000] in <filename unknown>:0 
  at minimal/core$eval__48806__48811.invoke () [0x00000] in <filename unknown>:0 
(6 stack frames omitted)
```

Per the documentation, this should return nil. But... It just throws an exception.

**Comment by LispEngineer** [comment]
Code that works for me:

```
(defn get-parent
  "Gets the parent GameObject of this GameObject, or nil if we're the top
   of the hierarchy. Works around a bug in Arcadia's 'parent' routine."
  ;; In C#:
  ;; go.transform.parent.gameObject does the trick, but .parent might be null
  ;; if it has no parent, so we have to double check.
  [go]
  (let [p (.. go transform parent)]
    (if (null-obj? p)
      nil
      (. p gameObject))))
```

**Comment by nasser** [comment]
Closed with #195
